### PR TITLE
fix font size and font family not working in create badges

### LINKED ISSUE #1949: font family and font size in create-badges not working properly
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behaviour:**
- long texts are not handled properly in create-badges, they just overflow
- changing font-family doesn't change font-family in the preview

**Expected behaviour:**
- long texts should automatically adjust fit the width of the badge.
- setting font sizes should work correctly
- setting font family should change font family in the preview

**Screenshots**
screenshot from 2019-01-16 19-22-30

screenshot from 2019-01-16 19-21-28

**Would you like to work on the issue?**
Yes, i'm working on this

### PR DESCRIPTION
**Pull request by YogeshSharma0201**
Fixes #1949 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:
- Added else condition to handle cases where <tspan> text node doesn't have any style attribute, i found some `<tspan>` inside `<text>` that don't have a style attribute.
```
                if text_style_detail is not None:
                    text_style_detail = text_style_detail.split(";")
                    for ind, i in enumerate(text_style_detail):
                        if i.split(':')[0] == 'fill':
                            text_style_detail[ind] = "fill:" + str(logo_fill)
                    text_style_detail = ";".join(text_style_detail)
                    t.set("style", text_style_detail)
                else:
                    t.set("style", "fill:" + str(logo_fill))
```
- Changing font-size and font-family works properly now.

### CODE REVIEW

**Comment by mariobehling** [comment]
Thanks. Please resolve conflicts.

**Comment by YogeshSharma0201** [comment]
Resolved. @mariobehling

File: frontend/app/templates/create-badges.hbs [hunk 1]
```diff
@@ -245,15 +245,12 @@
...
-        font_fn=defFont1Size font_ln=defFont2Size font_desg=defFont3Size font_org=defFont4Size font_sh=defFont5Size
-        fontcol_fn=defFontCol1
-        fontcol_ln=defFontCol2 fontcol_desg=defFontCol3 fontcol_org=defFontCol4 fontcol_sh=defFontCol5 defFont=defFont
-        defImageName=defImageName backImg=imageData backColor=backColor custLogoImage=custLogoImage
-        logoImageData=logoImageData
-        logoBackColor=logoBackColor logoFontColor=logoFontColor logo_text=logo_text defImage=defImage
-        custImage=custImage previewHeight=previewHeight
+        {{ user-component/preview fn=firstName ln=lastName org=organization sh=socialHandle desg=designation defColor=defFontColor
+        font_fn=defFont1Size font_ln=defFont2Size font_desg=defFont3Size font_org=defFont4Size font_sh=defFont5Size defFontType1=defFontType1
+        defFontType2=defFontType2 defFontType3=defFontType3 defFontType4=defFontType4 defFontType5=defFontType5 fontcol_fn=defFontCol1
```
Comment (abishekvashok) [inline frontend/app/templates/create-badges.hbs]: Pls don't change the formatting

**Comment by YogeshSharma0201** [comment]
@abishekvashok please review.

**Comment by abishekvashok** [comment]
Thanks

**Pull request merged**